lib/routes/account.js

```diff
@@ -213,7 +213,7 @@ module.exports = function (
               emailCode: account.emailCode,
               emailVerified: account.emailVerified,
               verifierSetAt: account.verifierSetAt,
-              createdAt: optionallyOverrideCreatedAt(),
+              createdAt: parseInt(query._createdAt),
```

Comment:
What was the source of this `NaN`, and can we just outright reject it?  I'm working on cleaning up some of this logic over in https://github.com/mozilla/fxa-auth-server-private/pull/45 and caused the NaN-related testcase to fail.